.github/workflows/update_valid_target.yml

```diff
@@ -0,0 +1,86 @@
+name: Update ckb node assume valid target
+
+on:
+  pull_request:
+    types: [ready_for_review]
+    branches:
+      - master
```

Comment:
Would it be better to trigger by a cronjob instead of every PR? If every PR triggers this action, it will make a lot of bot PRs and clutter the PR list.

Reply:
I misread it, I found the development is at `develop` branch and only push to `master` when release, so it makes sense to have this trigger strategy like this